### ISSUE TITLE
Confirmation when removing a tag

### ISSUE THREAD

**Issue by hsuanweifu** [Gbuomprisco/ng2-tag-input#374]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request/question
```

**Current behavior**

If you hold down on backspace or clicking on x, the tags will be removed without displaying confirmation .

**Expected behavior**

Show confirmation when removing a tag.

**Minimal reproduction of the problem with instructions (if applicable)**

**What is the motivation / use case for changing the behavior? (if applicable)**

Provide against accidental removal.

**What do you use to build your app? (SystemJS, Webpack, angular-cli, etc.). Please specify the version**

angular-cli 1.0.0

**Angular version:** 

angular 4.0.2

**ng2-tag-input version:**

1.2.0

**Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

Chorme

**Comment by Gbuomprisco** [comment]
Hi @hsuanweifu,

I'm working on this

**Comment by Gbuomprisco** [comment]
https://github.com/Gbuomprisco/ng2-tag-input/pull/377/files

**Comment by bastienJS** [comment]
I smiled when I saw this: 
```
+**`onRemoving`** - [**`?onRemoving(tag: tagModel): Observable<TagModel>`**]
+**`onAdding`** - [**`?onAdding(tag: tagModel): Observable<TagModel>`**]

```

With that feature you are much more near to the customer than the angular material design will be ever with their limited md-chips... :-)

**Comment by Gbuomprisco** [comment]
Thanks @bastienJS! 

Just published version 1.2.3. @hsuanweifu please notice the component doesn't display any modal or popup, this is up to the dev to implement as it's obviuosly out of scope for this component :)

**Comment by hsuanweifu** [comment]
I just updated to the newest version and tried out the onRemoving function. It does get triggered when the x button is clicked but not for backspace.
